src/validation/relationship_validator.py

```diff
@@ -43,9 +43,9 @@ def validate_relationship(relationship: Relationship, document: Document, spdx_v
         for message in messages:
             validation_messages.append(ValidationMessage(message, context))
 
-    if spdx_version != "2.3":
+    if spdx_version != "SPDX-2.3":
```

Comment:
might be good to put `"SPDX-2.3"` into an ENUM

Reply:
Actually, my plan was to convert `spdx_version` to a proper version field to then be able to properly implement conditions like "for all version below 2.3". See https://github.com/spdx/tools-python/issues/320